addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -168,6 +184,96 @@ export class MassMailingHtmlField extends HtmlField {
         await this._resetIframe();
     }
 
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Resize the given iframe so its height fits its contents and initialize a
+     * resize observer to resize on each size change in its contents.
+     * This also ensures the contents of the sidebar remain visible no matter
+     * how much we resize the iframe and scroll down.
+     *
+     * @private
+     */
+    _updateIframe() {
+        const iframe = this.wysiwyg.$iframe[0];
+        if (!iframe || !iframe.contentDocument) {
+            return;
+        }
+        const hasIframeChanged = !this.iframe || !this.iframe.contentDocument || iframe !== this.iframe;
+        this.iframe = iframe;
+        this._resizeMailingEditorIframe();
+
+        const iframeTarget = this.iframe.contentDocument.querySelector('#iframe_target');
```

Comment:
Not very important, but the JS lint configuration used in Odoo suggests always using `"` by default, unless unapplicable (this remark is valid for all new strings in this commit)

(you can enable the configuration for yourself using `odoo/addons/web/tooling/enable.sh` if you want)